code/controllers/subsystem/pai.dm

```diff
@@ -3,198 +3,168 @@ SUBSYSTEM_DEF(pai)
 
 	flags = SS_NO_INIT|SS_NO_FIRE
 
+	/// List of pAI candidates, including those not submitted.
 	var/list/candidates = list()
-	var/ghost_spam = FALSE
-	var/spam_delay = 100
+	/// Prevents a crew member from hitting "request pAI"
+	var/request_spam = FALSE
+	/// Prevents a pAI from submitting itself repeatedly and sounding an alert.
+	var/submit_spam = FALSE
```

Comment:
These are subsystem vars. Why does one person submitting a request once cause everyone to be ratelimited?